src/components/CreateOrUpdateRequest.svelte

```diff
@@ -30,7 +30,7 @@ const tomorrow = format(addDays(Date.now(), 1), 'yyyy-MM-dd')
 $: existingRequest = $requests.find(({ id }) => id === $params.requestId)
 $: initializeUpdates(existingRequest || defaults)
 $: isNew = !request.id
-$: eventId = request.event_id
+$: eventId = existingRequest?.meeting?.id
```

Comment:
not sure the first ? is needed, I could be wrong of course.

Reply:
It is. Found out by trial and error.